typings/index.d.ts

```diff
@@ -3031,13 +3031,11 @@ declare module 'discord.js' {
 
   type PartialTypes = 'USER' | 'CHANNEL' | 'GUILD_MEMBER' | 'MESSAGE' | 'REACTION';
 
-  interface PartialUser extends Partialize<User, 'bot' | 'flags' | 'locale' | 'system' | 'tag' | 'username'> {
+  interface PartialUser extends Omit<Partialize<User, 'bot' | 'flags' | 'locale' | 'system'>, "deleted"> {
```

Comment:
```suggestion
  interface PartialUser extends Partialize<User, 'bot' | 'flags' | 'locale' | 'system' | 'tag' | 'username' | 'deleted'> {
```
in-line with previous comment and not sure why you used Omit there, thats what the 2nd type param for `Partialize` is for

Reply:
The way I understand it is that the 2nd param only controls keys from the original type (`User`) and `Partialize` always adds its own properties on top without them being affected by the 2nd param in any way, no? So putting `deleted` there would only remove `deleted` from `User`, not `Partialize`.
The Base `Partialize` type is AND-ed with the rest here: https://github.com/discordjs/discord.js/blob/b0ab37ddc0614910e032ccf423816e106c3804e5/typings/index.d.ts#L2928-L2941

Reply:
oh right, facepalm there, mb

Reply:
quotes should be single quotes too, not double quotes

Reply:
Fixed